src/App.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div class="login_form_container">
+    <button
+      v-if="login === false"
```

Comment:
condition should be written in getter instead of <template>

Reply:
Do you mean [this getter](https://vuex.vuejs.org/guide/getters.html) in Vuex?